lib/validation.rb

```diff
@@ -99,6 +99,17 @@ def self.validate_postgres_flavor(flavor)
     end
   end
 
+  def self.validate_postgres_maintenance_window_start_at(maintenance_window_start_at)
```

Comment:
I think it's better to define `PostgresResource#validate` instead of defining this method:

```ruby
def validate
  super
  validates_includes(0..23, :maintenance_window_start_at, allow_nil: true, message: "must be between 0 and 23")
end
```